README.md

```diff
@@ -1,5 +1,7 @@
 ## Loopback in Five
 
+![logo](./tutorialPhotos/loopbackLogo.png)
+
 This is the GitHub repository that will show you how you can create a Node.js API in LoopBack in 5 easy minutes, and then how you can deploy that same API to Bluemix in 5 more minutes. 
 
 Click one of the links below to start your tutorial:
```

Comment:
or 
## Tutorials 
- Create an API with LoopBack in 5 minutes 
- Deploy your API with LoopBack in 5 minutes